jenkins-master/jobs/scripts/workspace/virtualenv/.gitignore

```diff
@@ -0,0 +1,10 @@
+virtualenv.egg-info
```

Comment:
I just read your justification in email. Still bothers me, but you have your reasons. A .zip file would be better.